src/components/LineChart/LineChart.jsx

```diff
@@ -226,11 +226,16 @@ const LineChart = createClass({
 		/**
 		 * An optional function used to format your y axis titles and data in the
 		 * tooltips. The first value is the name of your y field, the second value
-		 * is your post-formatted y value.
+		 * is your post-formatted y value, and the third value is your non-formatted
+		 * y-value.
 		 *
-		 * Signature: `(yField, yValueFormatted) => {}`
+		 * Signature: `(yField, yValueFormatted, yValue) => {}`
 		 */
 		yAxisTooltipFormatter: func,
+		/**
+		 * An optional function used to format data in the tooltips.
+		 */
+		yAxisTooltipDataFormatter: func,
```

Comment:
Would the `yAxisTooltipFormatter` signature be a better place for this? Only issue with that is, it wouldn't allow for formatting the two axes differently.